umode-api/src/lib.rs

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) 2022 by Rivos Inc.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#![no_std]
+
+//! Salus Hypervisor <-> U-mode API.
+//!
+//! This library contains data structures that are passed between
+//! hypervisor and umode.
```

Comment:
Expanding this comment to explain the calling convention would be helpful.

Reply:
Will do. Truth to be told I don't know where all this documentation goes.

Reply:
Done.